README.md

```
@@ -1,3 +1,140 @@
 # java-janggi
 
 장기 미션 저장소
+
+## 출력 포맷
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)